spec/systems/sessions_spec.rb

```diff
@@ -60,4 +60,48 @@
       end
     end
   end
+
+  describe 'logout' do
+    before(:each) do
```

Comment:
we don't use before each hook, https://compass.nimblehq.co/development/code-conventions/rspec/#fixtures-isolation

Reply:
Fixed in c4f4a0bf1125f19b406e95c8fe7bc507c28df2e5